packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -241,21 +239,6 @@ export class PendingStateManager implements IDisposable {
 		);
 		this.pendingMessages.shift();
 
-		// Processing part - Verify that there has been no data corruption.
-		// The clientSequenceNumber of the incoming message must match that of the pending message.
-		if (pendingMessage.clientSequenceNumber !== message.clientSequenceNumber) {
```

Comment:
Can we do some proxy test here? Like compare content of the pending message and incoming message to be the same (assuming it's safe to do such comparison)? Or maybe compare some safe to compare things (like internal runtime op type and path maybe - not full test, but it will capture any wrong doing over time)

Reply:
Proxy test as in have ContainerRuntime pass in a method to validate messages are the same? Such as checking `ContainerMessageType` and `IEnvelope.address`?

Reply:
Sorry, by proxy I meant something that is close by to old test, but done here. I.e. we have pending message and incoming message and it would be great to check they are the same (i.e. there is no bug in upper layer that results in us incorrectly assuming pending message was acked)

Reply:
@wes-carlson Do you think it's safe to check the stringified contents of `pendingMessage` and `message`? (Assuming the content isn't undefined)

Reply:
I think so